rubber_duck.py

```diff
@@ -106,15 +106,11 @@ def __init__(self,
         self._config = config
         self._metrics_handler = wrap_steps(metrics_handler)
         self._error_message_id = None
-        self.workflow_manager = workflow_manager
 
-        self.feedback_workflow = FeedbackWorkflow(
-            metrics_handler.record_feedback,
-            workflow_manager.send_event,
-            message_handler.send_message
-        )
+        self.start_feedback_workflow = step(start_feedback_workflow)
+
 
-    async def __call__(self, thread_id: int, engine: str, prompt: str, initial_message: Message, timeout=600):
+    async def __call__(self, thread_id: int, engine: str, prompt: str, initial_message: Message, timeout=60):#we changed this to 60 seconds for testing purposes
```

Comment:
Revert this change and remove the comment

Reply:
Done